src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/digitaltwinrepositoryprovisioningservice/models/__init__.py

```diff
@@ -0,0 +1,31 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: skip-file
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from .repository_key_request_properties import RepositoryKeyRequestProperties
```

Comment:
Please do not vendor an sdk within the CLI. Feel free to add it as a dependency in iot/setup.py. 

Please reach out if you have any questions

Reply:
Added in setup.py as dependencies